Makefile.PL

```diff
@@ -17,7 +17,8 @@ requires
   'HTML::Entities'           => 0,
   'IO::CaptureOutput'        => 0,
   'JSON::PP'                 => 0,
-  'JSON::RPC'                => 0.96,
+  'JSON::RPC'                => 1.00,
+  'JSON::RPC::Dispatch'      => 0,
```

Comment:
JSON::RPC-1.00 does not have JSON::RPC::Dispatch - it was introduced as JSON::RPC::Dispatch**er** in 1.00 and renamed to JSON::RPC::Dispatch in 1.00_02 because of a naming collision. Version 1.00_02 is marked as a developer release on the other hand. The best thing is probably to specify 1.01.

Also the JSON::RPC::Dispatcher line is useless with the bumped version number.